internal/models/users/users.go

```diff
@@ -281,3 +281,10 @@ func applyPreassignedRole(db *sql.DB, ctx context.Context, email string) (user_r
 
 	return user_roles.RoleFromString(roleString), nil
 }
+
+// Returns the number of users in the database
+func GetUserCount(db *sql.DB) (int, error) {
+	var count int
+	err := db.QueryRow(`SELECT COUNT(*) FROM users;`).Scan(&count)
```

Comment:
Do you know if counting on * instead of id increases the IO time? Because we could put an index on the ids.